cpu/atmega_common/periph/gpio.c

```diff
@@ -195,8 +281,70 @@ int gpio_init_int(gpio_t pin, gpio_mode_t mode, gpio_flank_t flank,
         return -1;
     }
 
-    /* not a valid interrupt pin */
+    /* not a valid interrupt pin. Set as pcint instead if pcints are enabled */
     if (int_num < 0) {
+	    /* If pin change interrupts are enabled, enable mask and interrupt */
+        #ifdef PCINT_NUM_BANKS
```

Comment:
We do not have an official RIOT coding style on indent of preprocessor, so treat this as a comment and feel free to just ignore it:

I personally do not favor to keep C language indent for CPP, as those are two different language and cannot reasonably be mixed. E.g. 

```C
switch(var) {
    case FOO:
        /* some code */
        break;
     #idef HAS_BAR
     case BAR:
         /* some code */
         break;
         #endif /* <-- two or one level of indent?
}
```

Therefore I personally favor to have no indent before the preprocessor macros. This also makes them more visible, as those look a bit out of place. This additional "spotlight" on c preprocessor code make overlooking them harder and - in my personal opinion - reading the code easier.

(But I again: Feel free to ignore this, there is no policy on that a.f.a.i.k.)

Reply:
no, you're right! :)